lib/template-dependencies.js

```diff
@@ -176,39 +176,42 @@ ${error.message}`
     });
   }
 
-  return elmJsonPromise.then(
-    (elmJsonCLI) =>
-      new Promise((resolve, reject) => {
-        const child = spawnAsync(elmJsonCLI, args, {
-          silent: true,
-          env: process.env
-        });
-        let stdout = '';
-        let stderr = '';
-
-        child.on('error', reject);
-
-        child.stdout.on('data', (chunk) => {
-          stdout += chunk.toString();
-        });
-
-        child.stderr.on('data', (chunk) => {
-          stderr += chunk.toString();
-        });
-
-        child.on('close', (code, signal) => {
-          if (code === 0) {
-            resolve(stdout);
-          } else {
-            const error = new Error(
-              `elm-json exited with ${exitReason(code, signal)}`
-            );
-            error.stderr = stderr;
-            reject(error);
-          }
-        });
-      })
-  );
+  return elmJsonPromise.then((elmJsonCLI) => {
+    // For CI elm-tooling returns the value of 0. CI environments
+    // are defined when the env var NO_ELM_TOOLING_INSTALL exists
+    if (elmJsonCLI === 0) return;
```

Comment:
`elmJsonCLI` is _always_ a string here (an absolute path to the `elm-json` executable), or am I confused?

Reply:
@lydell I think that this is an error on my side - there is something weird with this bunch of node_modules. I'm going to dig into this a bit more and will reopen this PR if needed,